.github/workflows/dactyl.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Dactyl build only on PR"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  build:
+    name: Build Dactyl Site
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Python Setup
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.7"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install Jinja2==2.11.3
+          pip install dactyl lxml
+          pip install --upgrade dactyl
+      # - name: Run Dactyl Link Checker
+      #   run: |
+      #     dactyl_link_checker -p /blog/
+      #
+      - name: PR Branch Name
+        uses: mdecoleman/pr-branch-name@1.0.0
+        id: branchname
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: GitHub Environment Variables Action
+        uses: FranzDiebold/github-env-vars-action@v1.2.1
+      - name: Get PR commit trigger SHA
+        uses: bhowell2/github-substring-action@v1.0.0
+        id: prsha
+        with:
+          value: ${{ github.event.pull_request.head.sha }}
+          length_from_start: 6
+      - name: Get short SHA6
+        id: slug
+        env:
+          PRSHA: ${{ steps.prsha.outputs.substring }}
+        run: echo "::set-output name=sha6::$(echo ${PRSHA} | cut -c1-6)"
+      - name: Get PR commit trigger SHA
+        uses: bhowell2/github-substring-action@v1.0.0
+        id: prsha7
+        with:
+          value: ${{ github.event.pull_request.head.sha }}
+          length_from_start: 7
+      - name: Get short SHA7
+        id: slug7
+        env:
+          PRSHA: ${{ steps.prsha7.outputs.substring }}
+        run: echo "::set-output name=sha7::$(echo ${PRSHA} | cut -c1-7)"
+      # TODO: Figure out how to prefix assets when not running at the root of the domain
+      # - name: Modify Docusaurus prefix in docusaurus.config.js
+      #   env:
+      #     SHA: ${{ steps.slug.outputs.sha6 }}
+      #     BRANCH: ${{ steps.branchname.outputs.branch }}
+      #     SLUG: ${{ env.GITHUB_REPOSITORY_NAME }}
+      #   run: |
+      #     sed -i "s@baseUrl: '/'@baseUrl: '/${SLUG}/review-apps/${BRANCH}/${SHA}/'@g" docusaurus.config.js
+      - name: Build and deploy to gh-pages
+        uses: sauloxd/review-apps@v1.3.3
+        with:
+          build-cmd: "dactyl_build"
```

Comment:
Some likely improvements we'll need (though they can come later):

- build all languages
    For now, that's like this:

        dactyl_build
        dactyl_build -t ja -o out/ja
- provide correct paths for "Edit" links in PRs. The [jenkins.sh](https://github.com/ripple/xrpl-dev-portal/blob/master/tool/jenkins.sh) script does this, though it gets the branch/PR info based on env vars set by the Jenkins PR builder, but you can get the idea